stdlib/kokorawr/kokorawr/functions/services/scores.js

```diff
@@ -1,39 +1,41 @@
 const lib = require('lib')({token: process.env.STDLIB_TOKEN});
 
+const winInc = {'bs': 31, 'c4': 23, 'ddg': [1, -5], 'mm': 37, 'rps': 3, 'ttt': 11}
```

Comment:
These numbers seem very arbitrary?

Reply:
Theyre the most random prime numbers I could find. Also, easier games win less points, whereas harder games that require thinking (like mm) win more. We can def change these tho

Reply:
So the thing is that these can be used to calculate the overall score, but the point of scores is to just keep a track of the win ratio between the two teams. This can be reported in the command though. Also these numbers should be mentioned in the herp command

Reply:
Ok so what is the win lose circumstance with ddg? Would win still be +1 and lose be -5, bc this wouldnt be a ratio anymore.

Reply:
yeah probably just keep it the same for ddg because it doesn't really make sense in any normal way since it's such an odd "game"